platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -631,13 +956,29 @@ odp_crypto_session_create(const odp_crypto_session_param_t *param,
 		}
 		break;
 	}
+	case ODP_CIPHER_ALG_AES_CBC:
+		/* Only AES-CBC and HMAC-SHA chained operation is supported */
+		if (param->auth_alg != ODP_AUTH_ALG_SHA1_HMAC &&
+		    param->auth_alg != ODP_AUTH_ALG_SHA256_HMAC) {
+			rc = -2;
```

Comment:
It is not clear to my why this needs to be the case. Maybe you could do an individual operation by just dropping the part of the result that you do not need?

The comment could be more helpful in explaining why individual operations are not supported. I can see from the code easily that only "chained" operation is supported, so the current comment does not add much. It would be much more useful to know _why_ cipher or auth alone cannot be supported.